docs/docs/start/docker_dev.md

```diff
@@ -115,6 +118,49 @@ This command launches the remaining containers:
 !!! success "Check Connection"
     Check that the server is running at [http://localhost:8000](http://localhost:8000). The server may take a few minutes to be ready.
 
+## Running commands in the container
+
+Using `docker compoase run [...]` commands creates a new container to run this specific command.
+This will eventually clutter your docker with many dead containers that take up space on the system.
+
+You can access the running containers directly with the following:
+```
+docker exec -it inventree-dev-server /bin/bash
+```
+You then run the following to access the virtualenv:
+```
+source data/env/bin/activate
+```
+
+This sets up a bash terminal where you can run `invoke` commands directly.
+
+!!! warning "Tests"
+    It is not recommended to run `invoke test` in your currently active inventree-dev-server container.
```

Comment:
Why?

Reply:
Severely increased test running times. I had one run that lasted 10k seconds 🙈😅

I'll add the reasoning to the doc.

Reply:
I still do not see the harm in it TBH. If it takes time it takes time, you can always kill the test suit.

Reply:
Well, the text works as you'd like then. It's saying that it's not recommended, but we don't say "don't ever do this"

Reply:
I do not agree with the warning about it. Running a few tests in the container is far faster than waiting for the CI you can specify which command should be run so there is no need to run everything.